ClientApp/src/app/components/modals/create-task-modal/create-task-modal.component.spec.ts

```diff
@@ -12,6 +12,7 @@ import { CheckboxModule } from 'primeng/checkbox';
 import { DialogModule } from 'primeng/dialog';
 import { InputNumberModule } from 'primeng/inputnumber';
 import { ToastModule } from 'primeng/toast';
+import { TaskTypePickerComponent } from '../../pickers/task-type-picker/task-type-picker.component';
```

Comment:
These types of imports should be from `src/app/components/pickers/...` to match the rest